st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -114,14 +114,26 @@ def __init__(self, pack, file_path, parameters=None, user=None, parent_args=None
     def run(self):
         action = self._get_action_instance()
         output = action.run(**self._parameters)
+        action_status = None
+        if type(output) is tuple and len(output) == 2:
+            action_status = output[0]
+            action_result = output[1]
+        else:
+            action_result = output
+
+        action_output = {"result": action_result}
 
         # Print output to stdout so the parent can capture it
         sys.stdout.write(ACTION_OUTPUT_RESULT_DELIMITER)
         print_output = None
-        try:
-            print_output = json.dumps(output)
-        except:
-            print_output = str(output)
+        if action_status is None:
+            print_output = json.dumps(action_output)
+        elif type(action_status) is bool:
+            action_output['status'] = action_status
+            print_output = json.dumps(action_output)
+        else:
+            raise Exception('Status should either be True or False.')
```

Comment:
Can you please make exception message a bit more explicit and user-friendly (e.g. "Status returned from the action must either be True or False").

In addition to that, let's please also add a test case for  this scenario (invalid status being returned from the Python action).